doc/manual/syntax.md

```diff
@@ -619,19 +648,182 @@ error: contract broken by a value.
 3
 ```
 
-Adding documentation can be done with `| doc < string >`. Examples:
+### Types
+
+The Nickel syntax mixes both terms and types in the same namespace. The
+following program is perfectly legal: `let value = Number -> (fun value label =>
+value) in ((fun x => x + 1) : value)`. See the
+[RFC002](https://github.com/tweag/nickel/blob/d723a3721c6b0fe9c4b856e889bb7211d6136665/rfcs/002-merge-types-terms-syntax.md)
```

Comment:
Is there any reason you are referencing the `RFC002` file from a particular commit instead of, say, `master`?

Reply:
But in general the RFC should be pretty much set in stone (if you need to evolve, better write a new one), so the benefit of being renaming/move/deletion resilient is probably higher than the risk of getting out of date?